### PR TITLE
Fix Qt not being able to create an XKB context

### LINKED ISSUE #104: Keyboard does not work on linux - Qt: Failed to create XKB context
Issue:

Keyboard does not work on linux in a PyQt application. `qtdiag` reports this
```
xkbcommon: ERROR: failed to add default include path 
Qt: Failed to create XKB context!
Use QT_XKB_CONFIG_ROOT environmental variable to provide an additional search path, add ':' as separator to provide several search paths and/or make sure that XKB configuration data directory contains recent enough contents, to update please see http://cgit.freedesktop.org/xkeyboard-config/ .
Qt 5.9.7 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 7.3.0) on "xcb" 
OS: Ubuntu 18.04.2 LTS [linux version 4.15.0-48-generic]

Architecture: x86_64; features: SSE2 SSE3 SSSE3 SSE4.1 SSE4.2 AVX
...
```

This does not occur for the `defaults` build of qt. I believe that the xkb path is configurable at build time and in the conda-forge build it is not set. I think the standard location for linux should be `/usr/share/X11/xkb`

Background info:
https://github.com/CadQuery/CQ-editor/issues/42
https://sft.its.cern.ch/jira/browse/SPI-1042

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# Name                    Version                   Build  Channel
alabaster                 0.7.10                   py36_1    conda-forge
asn1crypto                0.24.0                   py36_0    conda-forge
astroid                   1.6.5                    py36_0    conda-forge
atomicwrites              1.1.5                    py36_0  
attrs                     18.1.0                   py36_0  
babel                     2.6.0                    py36_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
blas                      1.0                         mkl  
bleach                    2.1.3                      py_0    conda-forge
ca-certificates           2019.3.9             hecc5488_0    conda-forge
cadquery                  0.5-SNAPSHOT              dev_0    <develop>
certifi                   2019.3.9                 py36_0    conda-forge
cffi                      1.11.5                   py36_0    conda-forge
chardet                   3.0.4                    py36_0    conda-forge
cloudpickle               0.5.3                      py_0    conda-forge
cqparts                   0.2.1                    pypi_0    pypi
cryptography              2.6.1            py36h1ba5d50_0  
curl                      7.64.1               hbc83047_0  
dbus                      1.13.6               he372182_0    conda-forge
decorator                 4.3.0                      py_0    conda-forge
docutils                  0.14                     py36_0    conda-forge
easyprocess               0.2.3                    pypi_0    pypi
entrypoints               0.2.3                    py36_1    conda-forge
expat                     2.2.6                he6710b0_0  
fontconfig                2.13.1               h65d0f4c_0    conda-forge
freeimageplus             3.17.0                        0    dlr-sc
freetype                  2.9.1                h6debe1e_4    conda-forge
future                    0.16.0                   py36_2    conda-forge
gettext                   0.19.8.1                      0    conda-forge
gl2ps                     1.3.8                         1    dlr-sc
glib                      2.58.3            hf63aee3_1001    conda-forge
gmp                       6.1.2                         0    conda-forge
gst-plugins-base          1.14.4            hdf3bae2_1001    conda-forge
gstreamer                 1.14.4            h66beb1c_1001    conda-forge
hdf4                      4.2.13                        0    conda-forge
hdf5                      1.10.2               hc401514_2    conda-forge
html5lib                  1.0.1                      py_0    conda-forge
icu                       58.2                          0    conda-forge
idna                      2.6                      py36_1    conda-forge
imagesize                 1.0.0                    py36_0    conda-forge
importlib-metadata        0.9                      pypi_0    pypi
intel-openmp              2019.0                      118  
ipykernel                 4.8.2                    py36_0    conda-forge
ipython                   6.4.0                    py36_0    conda-forge
ipython_genutils          0.2.0                    py36_0    conda-forge
isort                     4.3.4                    py36_0    conda-forge
jedi                      0.12.0                   py36_0    conda-forge
jeepney                   0.4                      py36_0  
jinja2                    2.10                     py36_0    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
jsoncpp                   1.8.4                h2d50403_0    conda-forge
jsonschema                2.6.0                    py36_1    conda-forge
jupyter_client            5.2.3                    py36_0    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
keyring                   15.1.0                   py36_0  
krb5                      1.16.1               h173b8e3_7  
lazy-object-proxy         1.3.1                    py36_0    conda-forge
libcurl                   7.64.1               h20c2e04_0  
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                         3    conda-forge
libgcc                    7.2.0                h69d50b8_2  
libgcc-ng                 8.2.0                hdf63c60_1  
libgfortran               3.0.0                         1  
libgfortran-ng            7.2.0                hdf63c60_3  
libiconv                  1.15                          0    conda-forge
libnetcdf                 4.6.1                h5e45101_3    conda-forge
libogg                    1.3.2                h470a237_1    conda-forge
libopenblas               0.2.20               h9ac9557_7  
libpng                    1.6.37               hbc83047_0  
libsodium                 1.0.16                        0    conda-forge
libssh2                   1.8.2                h1ba5d50_0  
libstdcxx-ng              8.2.0                hdf63c60_1  
libtheora                 1.1.1                h470a237_1    conda-forge
libtiff                   4.0.9                he6b73bb_1    conda-forge
libuuid                   2.32.1               h470a237_2    conda-forge
libvorbis                 1.3.5                h470a237_1    conda-forge
libxcb                    1.13                          0    conda-forge
libxml2                   2.9.9                h13577e0_0    conda-forge
logbook                   1.4.0                    pypi_0    pypi
logzero                   1.5.0                    pypi_0    pypi
lz4-c                     1.8.3                hfc679d8_0    conda-forge
markupsafe                1.0                      py36_0    conda-forge
mccabe                    0.6.1                    py36_0    conda-forge
mistune                   0.8.3                    py36_1    conda-forge
mkl                       2018.0.3                      1  
mkl_fft                   1.0.6            py36h7dd41cf_0  
mkl_random                1.0.1            py36h4414c95_1  
mock                      2.0.0            py36h3c5bf6c_0  
more-itertools            4.2.0                    py36_0  
nbconvert                 5.3.1                      py_1    conda-forge
nbformat                  4.4.0                    py36_0    conda-forge
ncurses                   6.1                  he6710b0_1  
numpy                     1.15.3           py36h1d66e8a_0  
numpy-base                1.15.3           py36h81de0dd_0  
numpydoc                  0.8.0                    py36_0    conda-forge
oce                       0.18.3               h1fac485_0    cadquery
openssl                   1.1.1b               h14c3975_1    conda-forge
packaging                 17.1                       py_0    conda-forge
pandoc                    2.2.1                         0    conda-forge
pandocfilters             1.4.2                    py36_0    conda-forge
parso                     0.2.1                      py_0    conda-forge
path-py                   12.0.1                   pypi_0    pypi
pbr                       4.0.3                    py36_0  
pcre                      8.42                 h439df22_0  
pexpect                   4.6.0                    py36_0    conda-forge
pickleshare               0.7.4                    py36_0    conda-forge
pip                       19.1                     py36_0  
pluggy                    0.6.0            py36hb689045_0  
prompt_toolkit            1.0.15                   py36_0    conda-forge
psutil                    5.4.6                    py36_0    conda-forge
ptyprocess                0.5.2                    py36_0    conda-forge
py                        1.5.3                    py36_0  
pycodestyle               2.4.0                    py36_0    conda-forge
pycparser                 2.18                     py36_0    conda-forge
pyflakes                  2.0.0                      py_0    conda-forge
pygments                  2.2.0                    py36_0    conda-forge
pylint                    1.9.2                    py36_0    conda-forge
pyopenssl                 18.0.0                   py36_0    conda-forge
pyparsing                 2.1.4                    py36_0  
pyqt                      5.9.2            py36h05f1152_2  
pyqtgraph                 0.10.0           py36h28b3542_3  
pysocks                   1.6.8                    py36_1    conda-forge
pytest                    3.6.0                    py36_1  
pytest-mock               1.10.0                   pypi_0    pypi
pytest-qt                 2.4.0                      py_0    conda-forge
pytest-xvfb               1.1.0                    pypi_0    pypi
python                    3.6.8                h0371630_0  
python-dateutil           2.7.3                      py_0    conda-forge
pythonocc-core            0.18.2           py36h39e3cac_0    cadquery
pytz                      2018.4                     py_0    conda-forge
pyvirtualdisplay          0.2.1                    pypi_0    pypi
pyzmq                     17.0.0                   py36_4    conda-forge
qt                        5.9.7                h52cfd70_1    conda-forge
qtawesome                 0.4.4                    py36_0    conda-forge
qtconsole                 4.4.4                      py_0    conda-forge
qtpy                      1.4.2            py36h8a2030e_0    conda-forge
readline                  7.0                  h7b6447c_5  
requests                  2.18.4                   py36_1    conda-forge
rope                      0.10.7                   py36_0    conda-forge
secretstorage             3.1.0                    py36_0  
setuptools                39.2.0                   py36_0    conda-forge
simplegeneric             0.8.1                    py36_0    conda-forge
sip                       4.19.8           py36hfc679d8_0    conda-forge
six                       1.11.0                   py36_1    conda-forge
smesh                     6.7.6                         7    pythonocc
snowballstemmer           1.2.1                    py36_0    conda-forge
sphinx                    1.7.4                    py36_0    conda-forge
sphinxcontrib-websupport  1.0.1                    py36_0    conda-forge
spyder                    3.3.1                    py36_1  
spyder-kernels            0.2.6                    py36_0  
sqlite                    3.28.0               h7b6447c_0  
tbb                       2019.1               h2d50403_0    conda-forge
testpath                  0.3.1                    py36_0    conda-forge
tinydb                    3.9.0.post1              pypi_0    pypi
tk                        8.6.8                hbc83047_0  
tornado                   5.0.2                    py36_0    conda-forge
traitlets                 4.3.2                    py36_0    conda-forge
typing                    3.6.4                    py36_0    conda-forge
urllib3                   1.22                     py36_0    conda-forge
vtk                       8.1.1           py36h4f1983a_204  
wcwidth                   0.1.7                    py36_0    conda-forge
webencodings              0.5                      py36_0    conda-forge
wheel                     0.31.0                   py36_0    conda-forge
wrapt                     1.10.11                  py36_0    conda-forge
xorg-libxau               1.0.8                         3    conda-forge
xorg-libxdmcp             1.1.2                         3    conda-forge
xz                        5.2.4                h14c3975_4  
zeromq                    4.2.5                         1    conda-forge
zipp                      0.3.3                    pypi_0    pypi
zlib                      1.2.11               h470a237_3    conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : cqgui
    active env location : /home/adam/anaconda2/envs/cqgui
            shell level : 2
       user config file : /home/adam/.condarc
 populated config files : 
          conda version : 4.6.14
    conda-build version : 3.0.22
         python version : 2.7.16.final.0
       base environment : /home/adam/anaconda2  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/adam/anaconda2/pkgs
                          /home/adam/.conda/pkgs
       envs directories : /home/adam/anaconda2/envs
                          /home/adam/.conda/envs
               platform : linux-64
             user-agent : conda/4.6.14 requests/2.21.0 CPython/2.7.16 Linux/4.15.0-48-generic ubuntu/18.04.2 glibc/2.27
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```
</details>

### PR DESCRIPTION
**Pull request by ccordoba12**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Fixes #104 
Fixes https://github.com/conda-forge/pyqt-feedstock/issues/52

@ocefpaf, please help us to recompile and upload a new Linux version.

### CODE REVIEW

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     matrix:
       win_cxx_compilervs2015vc14:
         CONFIG: win_cxx_compilervs2015vc14
-        CONDA_BLD_PATH: C:\\bld\\
+        CONDA_BLD_PATH: D:\\bld\\
```
Comment (ocefpaf) [inline .azure-pipelines/azure-pipelines-win.yml]: Please revert this change.
(We need to fix this in conda-smithy. I'll open an issue.)

File: recipe/build.sh [hunk 2]
```diff
@@ -149,7 +149,7 @@ if [[ ${HOST} =~ .*linux.* ]]; then
                 -qt-pcre \
                 -qt-xcb \
                 -qt-xkbcommon \
-                -xkb-config-root $PREFIX/lib \
+                -xkb-config-root /usr/share/X11/xkb \
```
Comment (ocefpaf) [inline recipe/build.sh]: @mingwandroid I would like to get your opinion here.

Comment (mingwandroid) [reply recipe/build.sh]: Seems OK? I think we had that.. I remember trying to use a pathlist with a : between but it doesn't work.

File: .azure-pipelines/azure-pipelines-win.yml (hunk 1 above)
Comment (ccordoba12) [reply .azure-pipelines/azure-pipelines-win.yml]: Done.

File: recipe/build.sh (hunk 2 above)
Comment (ocefpaf) [reply recipe/build.sh]: Build locally with this not. Hold on...

**Comment by ocefpaf** [comment]
Let's go with #106 just so we can have a 1-1 correspondence between the packaged uploaded and the recipe.

**Pull request closed without merging**